Pipfile

```diff
@@ -1,30 +1,38 @@
 [[source]]
+
 verify_ssl = true
 name = "pypi"
 url = "https://pypi.python.org/simple"
 
+
 [packages]
+
 mypy = "*"
 pylint = "*"
 requests = "*"
-beautifulsoup4 = "*"
+"beautifulsoup4" = "*"
```

Comment:
unnecesary "". Can you confirm it was intentionally 
Changed the `-` to `""`.

Reply:
Please see the answer [here](https://github.com/marius92mc/github-stargazers/pull/50#issuecomment-361176170).